docs/reference/app.md

```diff
@@ -1,10 +1,12 @@
 # app
 
-::: starlite.app.DEFAULT_OPENAPI_CONFIG
-
 ::: starlite.app.Starlite
     options:
-        show_source: false
         members:
             - __init__
+            - __call__
```

Comment:
this crashed for me when i tried it. working?

Reply:
![image](https://user-images.githubusercontent.com/20659309/186654340-7e2fd1ba-c187-42d3-a9f0-bc6306873d9f.png)